src/OrchardCore.Modules/OrchardCore.Users/Views/Items/ValidateUserTask.Fields.Edit.cshtml

```diff
@@ -8,6 +8,6 @@
 
 <fieldset class="form-group" asp-validation-class-for="Roles">
     <label asp-for="Roles">@T["Roles"]</label>
-    @await Component.InvokeAsync("SelectRoles", new { selectedRoles = Model.Roles, htmlName = Html.NameFor(m => m.Roles) })
+    @await Component.InvokeAsync("SelectRoles", new { selectedRoles = Model.Roles, htmlName = Html.NameFor(m => m.Roles), except = new[] { "Anonymous", "Authenticated" }  })
```

Comment:
Can we use static field or property for the `except` instead of newing up in each render for the field